src/components/org/OrgAdmin/AboutSection/Curriculum/index.tsx

```diff
@@ -1,37 +1,51 @@
-import { Chip } from '@sopt-makers/ui';
-
-import { PART_LIST } from '@/utils/org';
+import { useState } from 'react';
 
+import PartCategory from '../../PartCategory';
 import { StInput, StTitle, StWrapper } from '../style';
-import {
-  StChipWrapper,
-  StContentWrapper,
-  StItem,
-  StList,
-  StWeek,
-} from './style';
+import { StContentWrapper, StItem, StList, StWeek } from './style';
 
 const Curriculum = () => {
-  const CURRICULUM = ['', '2주차', '3주차', '4주차', '', '', '', ''];
+  const [selectedPart, setSelectedPart] = useState('기획');
+  const [curriculum, setCurriculum] = useState<Record<string, string[]>>({
+    기획: Array.from({ length: 8 }, () => ''),
+    디자인: Array.from({ length: 8 }, () => ''),
+    안드로이드: Array.from({ length: 8 }, () => ''),
+    iOS: Array.from({ length: 8 }, () => ''),
+    웹: Array.from({ length: 8 }, () => ''),
+    서버: Array.from({ length: 8 }, () => ''),
+  });
```

Comment:
```suggestion
  const createInitialCurriculum = () =>
    ['기획', '디자인', '안드로이드', 'iOS', '웹', '서버'].reduce(
      (acc, part) => {
        acc[part] = Array(8).fill('');
        return acc;
      },
      {} as Record<string, string[]>,
    );

  const [curriculum, setCurriculum] = useState(createInitialCurriculum);
```

이런 식으로 반복을 좀 줄일 수도 있을 거 같아요~

Reply:
좋아요!

Reply:
한번 초기화되는 친구라 굳이 함수화할 필요 없을 것 같아서 아래처럼 해주었습니다 ! ! 
```ts
  const initialCurriculum = PART_LIST.reduce(
    (acc, part) => {
      acc[part] = Array.from({ length: 8 }, () => '');
      return acc;
    },
    {} as Record<string, string[]>,
  );

  const [curriculum, setCurriculum] = useState(initialCurriculum);
```

Reply:
넵 확인했습니다! 고생하셨습니다~ :)